test/testcmdlineparser.cpp

```diff
@@ -106,6 +106,11 @@ class TestCmdlineParser : public TestFixture {
         TEST_CASE(stdcpp11);
         TEST_CASE(stdunknown);
         TEST_CASE(platform);
+        TEST_CASE(platform2);
```

Comment:
Could you name the test function according to the platform file tested? e.g. platform_win64 etc.

Reply:
Those will probably go away soon anyways because of the incremental nature...

Reply:
Ok